package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -509,6 +509,12 @@ dhcp_boot_add() {
 	dhcp_option_add "$cfg" "$networkid" "$force"
 }
 
+eval_ipcalc() {
+	local output
+	if output="$(ipcalc.sh "$@")"; then
+		eval "$output"
+	fi
```

Comment:
Still need to forward the return code of ipcalc somehow. Possibly
```suggestion
	else
		return 1
	fi
```

Should maybe also use this for packages/isc-dhcp (or another way of testing ipcalc.sh return), because it does use `$START` and `$END` (I missed this first look, sorry) https://github.com/openwrt/packages/blob/e257404154a78a45bba3919a3b1bf3c2b9b1aa72/net/isc-dhcp/files/dhcpd.init#L413